src/style.css

```diff
@@ -1,3 +1,76 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, system-ui;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  color: #fff;
+}
+
+.image {
+  display: block;
+  line-height: 0;
+}
+
+.nav .list {
+  display: flex;
+  list-style: none;
+}
+
+.list_item {
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. It seems `.list_item` should be `.nav__item` to follow BEM methodology and to be consistent with the meaning of the content.